.gitignore

```diff
@@ -142,3 +142,6 @@ cython_debug/
 
 # Project specific
 test.py
+
+# Env variables
+dev.env
```

Comment:
Personally I like this as '.env', e.g. with a dot so it's hidden by default and als some kind of standard in a lot of projects I have on my plate here. Not sure if it is an actual standard, though.